RegulatedPurePursuitController:
  desired_linear_vel: 0.3
  lookahead_dist: 0.5
  min_lookahead_dist: 0.3
  max_lookahead_dist: 0.7
  lookahead_time: 1.5
  rotate_to_heading_angular_vel: 0.3
  transform_tolerance: 0.1
  use_velocity_scaled_lookahead_dist: false
  min_approach_linear_velocity: 0.05
  approach_velocity_scaling_dist: 0.6
  use_collision_detection: true
  max_allowed_time_to_collision_up_to_carrot: 1.0
  use_regulated_linear_velocity_scaling: true
  use_fixed_curvature_lookahead: false
  curvature_lookahead_dist: 0.25
  use_cost_regulated_linear_velocity_scaling: true
  regulated_linear_scaling_min_radius: 0.9
  regulated_linear_scaling_min_speed: 0.25
  use_rotate_to_heading: true
  allow_reversing: false
  rotate_to_heading_min_angle: 0.5
  max_angular_accel: 0.3
  max_linear_accel: 2.0
  max_robot_pose_search_dist: 10.0
  theta_stopped_vel: 1.0
  trans_stopped_vel: 0.02
  goal_dist_tol: 0.03
  angle_tol: 0.03
  max_linear_vel: 0.3
  max_angular_vel: 0.3
  cost_scaling_dist: 0.5
  cost_scaling_gain: 1.0
  inflation_cost_scaling_factor: 3.0
  interpolate_curvature_after_goal: false
  k: 6.0
  min_turning_radius: 0.2
  use_vector_pure_pursuit: false
